tools/scoring/tests/parse_config_test.py

```diff
@@ -57,6 +60,28 @@ def testAppendTypes(self, mock_print):
     ]
     mock_print.assert_has_calls(calls)
 
+  def testMatchReportingEntities(self):
+    proposed = validator.Deserialize(
+        ['tests/fixtures/files/proposed/match_reporting_entities.yaml'])[0]
+    solution = validator.Deserialize(
+        ['tests/fixtures/files/solution/match_reporting_entities.yaml'])[0]
+
+    matches = parse_config.ParseConfig.match_reporting_entities(
+        proposed=proposed, solution=solution)
+
+    self.assertEqual(len(proposed), 4)
+    self.assertEqual(len(solution), 4)
+    self.assertEqual(len(matches),
+                     2)  # number of valid reporting entities in solution
```

Comment:
I can see only one match here: `2599571827844401` which one is the second match?

Reply:
"matches" is a bit of a misnomer as it's a list of valid reporting entities and their matches _if they exist_, hence the comment. Would "pairs" be more intuitive?

Reply:
Further thoughts: should this just be a dict with IDs to be referenced against the original "proposed" and "solution" dicts? It would mean two additional lookups for each pair but the signature would be much clearer.